htdocs/core/modules/commande/doc/pdf_eratosthene.modules.php

```diff
@@ -864,9 +864,9 @@ public function write_file($object, $outputlangs, $srctemplatepath = '', $hidede
 				}
 
 				// Add terms to sale
-				if (getDolGlobalString("MAIN_INFO_ORDER_TERMSOFSALE") && getDolGlobalInt('MAIN_PDF_ADD_TERMSOFSALE_ORDER')) {
-					$termsofsalefilename = getDolGlobalString('MAIN_INFO_ORDER_TERMSOFSALE');
-					$termsofsale = $conf->order->dir_output.'/'.$termsofsalefilename;
+				if (getDolGlobalString("MAIN_INFO_SOCIETE_TERMSOFSALE") && getDolGlobalInt('MAIN_PDF_ADD_TERMSOFSALE_ORDER')) {
+					$termsofsalefilename = getDolGlobalString('MAIN_INFO_SOCIETE_TERMSOFSALE');
```

Comment:
On devleop branch, the feature to add term of sale allows to define a different file for each type of object.
Si we must have here MAIN_INFO_ORDER_TERMSOFSALE for order, MAIN_INFO_PROPAL_TERMSOFSALE for proposal, etc...